src/packages/utils/src/utils/min-max.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * As Math.min/Math.max for number/bigint.
```

Comment:
I looked up the definition of Math.max on MDN, and it says `returns the largest of the numbers given as input parameters, or -Infinity if there are no parameters".

Obviously we aren't returning `Infinity`, since we just let `reduce` throw, but I like the rest of the description.

So maybe the JSDoc for this could be something like:

```
/**
 * Returns the largest of the numbers/bigints given as input parameters. Throws if no values are given.
 * @param numbers One or more numbers and/or bigints among which the largest value will be selected and returned.
 */
```

And same for `min`

Reply:
:+1: